internal/hud/webview/convert.go

```diff
@@ -6,23 +6,20 @@ import (
 	"strings"
 
 	"github.com/golang/protobuf/ptypes"
-
-	"github.com/tilt-dev/tilt/internal/k8s"
-
-	"github.com/tilt-dev/tilt/pkg/apis"
-
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/tilt-dev/tilt/internal/cloud/cloudurl"
+	"github.com/tilt-dev/tilt/internal/k8s"
 	"github.com/tilt-dev/tilt/internal/store"
+	"github.com/tilt-dev/tilt/internal/store/k8sconv"
+	"github.com/tilt-dev/tilt/pkg/apis"
```

Comment:
Not important, but did you do some manual re-arranging to get the imports "correctly" sorted? I'm sorta surprised `goimports` hadn't placed them here before, so just wondering if there was magic involved

Reply:
there was some manual reordering :grimacing:

ya, might dig into sometime why goimports doesn't do this on its own

Reply:
I'm a bit late to the party but this has bugged me for a long time: https://github.com/golang/go/issues/20818

FWIW when I notice this in a file I just delete all the blank lines in the import block and then goimports will do the right thing.